### DISCUSSION TITLE
The selected scanner is inactive. EPSON L395

### DISCUSSION THREAD

**Discussion by RuHor** [cyanfish/naps2#255] [Support]
Hello, I installed version 7.2.0 and also version 7.2.1 but with the scanner of my EPSON L395 network printer I get this error: the selected scanner is inactive.

If I use other software to scan documents it works without a problem. Add the software to the firewall and still get the same message.

I checked to see if any log was generated with the error, but it does not exist.

**Comment by cyanfish** [comment]
What driver option (TWAIN, WIA, ESCL) are you using? Have you tried the other options?

**Reply by RuHor** [reply]
Very kind to answer, I use TWAIN, I tried the other options, but it doesn't work, only the scanner is detected in TWAIN, previously the application worked but with the latest versions I can no longer make the scanner work with NAPS2.

**Reply by cyanfish** [reply]
If you go to Advanced in your profile settings, you could try different options for TWAIN Implementation. But it sounds like it's probably an issue with the manufacturer driver, you can try re-installing from the manufacturer website.

**Reply by RuHor** [reply]
My printer's scanner is on the network and when trying to use it generates the "scanner inactive" error, I tried many various options within the software but it does not work if it is over the network when trying to use it, but if I connect it with a direct USB to the printer it works there . It is somewhat strange that it does not work on a network but I think that when I need to use it I will have to install a 3 meter cable that I have to work without problem.
